proxy/envoy/policy.go

```diff
@@ -18,39 +18,58 @@
 package envoy
 
 import (
+	"sort"
 	"strings"
 
 	"istio.io/manager/model"
 	proxyconfig "istio.io/manager/model/proxy/alphav1/config"
 )
 
 func insertMixerFilter(listeners []*Listener, context *ProxyContext) {
-	if context.MeshConfig.MixerAddress != "" {
-		// join IPs with a comma
-		ips := make([]string, 0)
-		for ip := range context.Addrs {
-			ips = append(ips, ip)
-		}
-		id := strings.Join(ips, ",")
+	if context.MeshConfig.MixerAddress == "" {
+		return
+	}
+
+	// join IPs with a comma
+	ips := make([]string, 0)
```

Comment:
Same thing for service instances except there instances are per-port (so you'd get several structs with same service)